core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1131,7 +1131,7 @@ public String toString() {
           .build();
   public static final PropertyKey WORKER_NETWORK_NETTY_ASYNC_CACHE_MANAGER_THREADS_MAX =
       new Builder(Name.WORKER_NETWORK_NETTY_ASYNC_CACHE_MANAGER_THREADS_MAX)
-          .setDefaultValue(512)
+          .setDefaultValue(8)
```

Comment:
Does it make sense to use a  dynamic number that's proportional to the number of cores?
https://stackoverflow.com/questions/4759570/finding-number-of-cores-in-java

Reply:
it is tricky to argue # cores is the magic number here, but having it related to #cores seems reasonable.
I propose to #cores / 2 . Remember that this is the background service with best effort. Depending on the application, I don't think use all CPU resources on this task is a good idea though.

Reply:
unfortunately, supporting runtime-default value is not well supported before https://github.com/Alluxio/alluxio/pull/6866, which is merged to master.
That PR is too big to be cherry-picked to 1.7 branch IMO.

Do you feel strong about having this? @yupeng9

Reply:
I see. I'm okay with keeping this a fixed value for 1.7, but improving it in the master branch.

Reply:
agree. I will make a patch in master to improve this value setting